packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -118,3 +120,15 @@ export function generateConfigurations(
 	}
 	return generatePairwiseOptions<Record<string, ConfigTypes>>(overrides, seed);
 }
+
+export function getOptionOverride(
+	testConfig: ILoadTestConfig | undefined,
+	driverType: TestDriverTypes,
+	endpoint: string | undefined,
+): OptionOverride | undefined {
+	const driverEndpointOverride = `${driverType}-${endpoint}`;
+	return (
```

Comment:
A comment descirbing the behavior would be helpful.

Reply:
this doesn't support merging of driver and endpoint, it's all or nothing

Reply:
Yeah, I didn't want to cause any unexpected side effects.

Reply:
They way the configuration options are written are all or nothing